example_regression_2.py

```diff
@@ -0,0 +1,103 @@
+"""Load a saved regression model, make predictions of various types and save them to a csv or database.
+
+Please use this example to learn about healthcareai before moving on to the next example.
+
+If you have not installed healthcare.ai, refer to the instructions here:
+  http://healthcareai-py.readthedocs.io
+
+To run this example:
+  python3 example_regression_2.py
+
+This code uses the DiabetesClinicalSampleData.csv source file.
+"""
+import pandas as pd
+
+import healthcareai.common.file_io_utilities as io_utilities
+import healthcareai.common.write_predictions_to_database as hcaidb
+
+
+def main():
+    # Load data from a sample .csv file
+    prediction_dataframe = pd.read_csv('healthcareai/tests/fixtures/DiabetesClinicalSampleData.csv', na_values=['None'])
+
+    # Load data from a MSSQL server: Uncomment to pull data from MSSQL server
+    # server = 'localhost'
+    # database = 'SAM'
+    # query = """SELECT *
+    #             FROM [SAM].[dbo].[DiabetesClincialSampleData]
+    #             WHERE SystolicBPNBR is null"""
+    #
+    # engine = hcaidb.build_mssql_engine(server=server, database=database)
+    # prediction_dataframe = pd.read_sql(query, engine)
+
+    # Drop columns that won't help machine learning
+    columns_to_remove = ['PatientID']
+    prediction_dataframe.drop(columns_to_remove, axis=1, inplace=True)
+
+    # Load the saved model and print out the metrics
+    trained_model = io_utilities.load_saved_model('2017-05-31T15-55-55_regression_LinearRegression.pkl')
```

Comment:
The file name of the saved model created by regression example 1 depends on the time the file was created so will be different every time the regression example 1 is run.  Unlike classification example 1, regression example 1 does create a file similar to the one referenced above.  Since the file name is different each time the code is run, the files can start to pile up if the example is run many times.